kubernetes/test/test_k8sclient.py

```diff
@@ -0,0 +1,204 @@
+# -*- coding: utf-8 -*-
```

Comment:
the test folder will be removed and re-generated by scripts/update-client.sh. You need to either change the script there to keep this file or move this file to another folder. maybe put it in kubernetes/e2e_test folder or something like that? Also would be nice if we can have an script in scripts/ folder to deploy a minikube and run this against it.